### PR TITLE
[OSPO Book]  minor edits in chapter 2 and 3

### PR DESCRIPTION
**Pull request by anajsana**

### CODE REVIEW

File: ospo-book/content/en/02-chapter.md [hunk 1]
```diff
@@ -4,19 +4,25 @@ status: Completed
...
+Organizations of various types—including end-user companies, software vendors, universities, and public administrations—maintain a relationship with open source. To responsibly manage not only software but also hardware, content, and other aspects of technology, organizations must engage with open source. This involves finding ways to integrate such culture and operations into their IT strategy and technology and AI stacks. Establishing an organizational structure is a crucial first step to solidify commitment. This is where an Open Source Program Office (OSPO) becomes key. It serves as a means for organizations to support their objectives and address challenges related to open source.
 
-Below are common situations where an organization, which needs to manage open source integral to its digital/IT strategy and infrastructure, can encounter issues. When ignored or neglected, these issues can lead to mid-term to long-term innovation bottlenecks and security vulnerabilities.
+> 💡 OSPOs are all about Integration, not Isolation
```
Comment (UlisesGascon) [inline ospo-book/content/en/02-chapter.md:19]: I love these quotes!

**Review by UlisesGascon — APPROVED** [review]
LGTM! Thanks for all the refinement and additions @anajsana!

File: ospo-book/content/en/03-chapter.md [hunk 2]
```diff
@@ -59,6 +59,10 @@ The structure used in this book to represent these areas is shaped as a flower d
 
 #### Creating and Implementing an Open Source Strategy
 
+> If an organization's primary objective is profitability, customer satisfaction becomes a linchpin in the corporate strategy – a focal point for both CEOs and CFOs. For individuals in Open Source Program Offices, effectively communicating the open source strategy to C-level executives demands a keen understanding of the industry landscape and alignment with the key considerations of CEOs and CFOs. This alignment necessitates a clear comprehension of the overarching corporate strategy and identifying technologies within the open-source realm that can propel the organization toward its strategic objectives
+>
+> Victor Lu - [Strategy - End Game for FINOS Maturity Model](https://docs.google.com/presentation/d/1jJtR6-fvU-dCrGq_gTm4P1Awv90oCu4RClj1919970A/edit?usp=sharing)
```
Comment (CsatariGergely) [inline ospo-book/content/en/03-chapter.md:64]: Is this a quote from Victor's presentation? I could not find the actual quote in the slides, maybe it was mentioned when the presentation we delivered? In this case we should link the video.

File: ospo-book/content/en/03-chapter.md [hunk 3]
```diff
@@ -173,13 +177,13 @@ There is a lack of consistency in how open source understanding and value is per
...
-* [Open Source maturity model](https://www.finos.org/blog/open-source-maturity-model-launch)
+- [FINOS open source maturity model](https://www.finos.org/blog/open-source-maturity-model-launch)
+- [Porsche Open Source Website](https://opensource.porsche.com/)
+- [OSPO maturity model](https://linuxfoundation.org/tools/the-evolution-of-the-open-source-program-office-ospo/)
+- [OSPO 101 training module - OSPO and your organization](https://github.com/todogroup/ospo-career-path/tree/main/OSPO-101/module3)
+- [Setting up an open source strategy](https://todogroup.org/resources/guides/setting-an-open-source-strategy/).
+- [Participating in open source communities](https://todogroup.org/resources/guides/participating-in-open-source-communities/)
+- [A guide to outbound open source software](https://todogroup.org/resources/guides/a-guide-to-outbound-open-source-software/)
+- [A deep dive into OSPO](https://www.linuxfoundation.org/research/a-deep-dive-into-open-source-program-offices)
+- [Open Source maturity model](https://www.finos.org/blog/open-source-maturity-model-launch)
```
Comment (CsatariGergely) [inline ospo-book/content/en/03-chapter.md:188]: This is a duplicate of line 180

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: ospo-book/content/en/03-chapter.md (hunk 3 above)
Comment (koozz) [reply ospo-book/content/en/03-chapter.md:188]: You can add a quick suggestion to remove it. With the `[±]` "Add a suggestion <ctrl+g>" button.

```suggestion
```

File: ospo-book/content/en/03-chapter.md (hunk 2 above)
Comment (anajsana) [reply ospo-book/content/en/03-chapter.md:64]: That comes from a summary of the slides written by Victor himself 🙂 He shared this during one of the past calls

File: ospo-book/content/en/03-chapter.md (hunk 3 above)
Comment (anajsana) [reply ospo-book/content/en/03-chapter.md:188]: thank you!

**Pull request merged**